docs/core/rid-catalog.md

```diff
@@ -1,136 +1,176 @@
+- Don't build RIDs programmatically.
+- Use RIDs that are already defined for the platform.
+- The RIDs need to be specific, so don't assume anything from the actual RID value.
 
 ## Using RIDs
-In order to use RIDs, you have to know which RIDs there are. New RIDs are added regularly to the platform. 
-For the latest version, please check the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
 
-> [!NOTE]
-> We are working towards getting this information into a more interactive form. 
-> When that happens, this page will be updated to point to that tool and/or its usage documentation. 
+To be able to use RIDs, you have to know which RIDs exist. New values are added regularly to the platform.
+For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
+
+.NET Core 2.0 SDK introduces the concept of portable RIDs. They are new values added to the RID graph that aren't tied to a specific version or OS distribution. They're particularly useful when dealing with the multiple Linux distros.
+
+The following list shows the most common RIDs used for each OS. It doesn't cover `arm` or `corert` values.
 
 ## Windows RIDs
 
-* Windows 7 / Windows Server 2008 R2
-    * `win7-x64`
-    * `win7-x86`
-* Windows 8 / Windows Server 2012
-    * `win8-x64`
-    * `win8-x86`
-    * `win8-arm`
-* Windows 8.1 / Windows Server 2012 R2
-    * `win81-x64`
-    * `win81-x86`
-    * `win81-arm`
-* Windows 10 / Windows Server 2016
-    * `win10-x64`
-    * `win10-x86`
-    * `win10-arm`
-    * `win10-arm64`
+- Portable
+  - `win-x86`
+  - `win-x64`
+- Windows 7 / Windows Server 2008 R2
+  - `win7-x64`
+  - `win7-x86`
+- Windows 8 / Windows Server 2012
+  - `win8-x64`
+  - `win8-x86`
+  - `win8-arm`
+- Windows 8.1 / Windows Server 2012 R2
+  - `win81-x64`
+  - `win81-x86`
+  - `win81-arm`
+- Windows 10 / Windows Server 2016
+  - `win10-x64`
+  - `win10-x86`
+  - `win10-arm`
+  - `win10-arm64`
 
 ## Linux RIDs
 
-* Red Hat Enterprise Linux
-    * `rhel.7-x64`
-* Ubuntu
-    * `ubuntu.14.04-x64`
-    * `ubuntu.14.10-x64`
-    * `ubuntu.15.04-x64`
-    * `ubuntu.15.10-x64`
-    * `ubuntu.16.04-x64`
-    * `ubuntu.16.10-x64`
-* CentOS
-    * `centos.7-x64`
-* Debian
-    * `debian.8-x64`
-* Fedora
-    * `fedora.23-x64`
-    * `fedora.24-x64`
-* OpenSUSE
-    * `opensuse.13.2-x64`
-    * `opensuse.42.1-x64`
-* Oracle Linux
-    * `ol.7-x64`
-    * `ol.7.0-x64`
-    * `ol.7.1-x64`
-    * `ol.7.2-x64`
-* Currently supported Ubuntu derivatives 
-    * `linuxmint.17-x64`
-    * `linuxmint.17.1-x64`
-    * `linuxmint.17.2-x64`
-    * `linuxmint.17.3-x64`
-    * `linuxmint.18-x64`
+- Portable
+  - `linux-x64`
+- CentOS
+  - `centos-x64`
+  - `centos.7-x64`
+- Debian
+  - `debian-x64`
+  - `debian.8-x64`
+- Fedora
+  - `fedora-x64`
+  - `fedora.23-x64`
```

Comment:
I think we can remove fedora.23 and opensuse.13.2 since they've been "end of life'd".